doc/src/usage.md

```diff
@@ -32,6 +32,13 @@ target, see [Per Target options](#per-target-options) for details.
 [`INTERFACE`]: https://cmake.org/cmake/help/latest/command/add_library.html#interface-libraries
 [target_link_libraries]: https://cmake.org/cmake/help/latest/command/target_link_libraries.html
 
+### Install crate and headers with `corrosion_install`
```

Comment:
I would add an "Experimental: " here. The functionality is not complete, and we don't have tests (yet), so I would expect a few things to be broken in the beginning.

Reply:
Fixed. I also added the `EXPERIMENTAL` warning inside the function header found on other internal functions.